cypress/support/gutenberg/commands/controls.js

```diff
@@ -199,6 +203,99 @@ function toggleControl( name, value, options = {} ) {
 		} )
 }
 
+/**
+ * Command for adjusting the text control.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function textControl( name, value, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+	} = options
+
+	beforeAdjust( name, value, options )
+	cy.getBaseControl( name, { isInPopover } )
+		.find( 'input.components-text-control__input' )
+		.type( `{selectall}${ value }`, { force: true } )
+}
+
+/**
+ * Command for typing into a text area control.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function textAreaControl( name, value, options = {} ) {
+	const {
+		isInPopover = false,
+		beforeAdjust = () => {},
+	} = options
+
+	beforeAdjust( name, value, options )
+	cy.getBaseControl( name, { isInPopover } )
+		.find( 'textarea.components-textarea-control__input' )
+		.type( `{selectall}${ value }`, { force: true } )
+}
+
+/**
+ * Command for changing the block style control in native blocks.
+ *
+ * @param {string} name
+ * @param {*} value
+ * @param {Object} options
+ */
+function stylesControl( name, value, options = {} ) {
+	const {
+		beforeAdjust = () => {},
+	} = options
+
+	cy.get( '.block-editor-block-styles' )
+		.find( `div.block-editor-block-styles__item[aria-label=${ value }]` )
+		.invoke( 'attr', 'class' )
+		.then( classNames => {
+			const parsedClassNames = classNames.split( ' ' )
+			if ( value && ! parsedClassNames.includes( 'is-active' ) ) {
```

Comment:
do we still need to check this?

Reply:
No need to check na pala! I'll change it

Reply:
Thank you!